docs/features/hardware-offload/ovs-doca.md

```diff
@@ -0,0 +1,229 @@
+# OVS-DOCA
+
+## Introduction
+
+OVS supports Hardware Offload features which allows to offload OVS data-plane to the NIC while maintaining OVS control-plane unmodified. It is using SR-IOV technology with VF representor host net-device.
```

Comment:
Apologies for the confusing wording. 

What I meant to suggest is that all the OVS client side api's and tools (ovs-vsctl, ovs-ofctl, ovs-appctl, openflow configurations etc) are the same [control plane], while the packet processing itself [dataplane] is done in dedicated networking specific hardware instead of software (x86).

I can reword it if it makes better sense.